### DISCUSSION TITLE
DietPi-Software | Memos

### DISCUSSION THREAD

**Discussion by itsdoublearon** [MichaIng/DietPi#6729] [Software Request]
Hi there, I would like to ask if Memos could be added to the software lineup please..
https://github.com/usememos/memos

**Comment by Joulinar** [comment]
Looks like Docker only. Usually we don't add Docker apps.

**Comment by MichaIng** [comment]
Should be possible to deploy with `go install` as well. As it is a single binary without any other service dependencies, Docker looks like an unnecessary overhead to me.
